lib/elixir_newbie/blog/post.ex

```diff
@@ -1,6 +1,6 @@
 defmodule ElixirNewbie.Blog.Post do
   @enforce_keys [:id, :title, :cover_image, :body, :description, :tags, :date]
-  defstruct @enforce_keys ++ [:livebook_url]
+  defstruct @enforce_keys ++ [:livebook_url, :author]
```

Comment:
How about `:author_name` as the key? I think we're going to add more author information, such as a profile picture.

Reply:
Makes sense, i will change